dt-contacts/base-setup.php

```diff
@@ -661,10 +661,95 @@ public function add_comm_channel_comment_section( $sections, $post_type ){
                     'label' => esc_html( $channel_option['name'] ?? $channel_key )
                 ];
             }
+
+            // Extract custom comment types.
+            $custom_field_options  = dt_get_option( 'dt_field_customizations' );
```

Comment:
@kodinkat I think this would be simpler if it wasn't save in the custom fields array. A comment type isn't a custom field.
Lets move it to it's own option that soly stores custom comments types. Then merge those options  over the filtered ones.